README.md

```diff
@@ -8,6 +8,29 @@ Ansible role for installing and configuring an
 [OpenVPN](https://openvpn.net) server.  This role also enables IPv4
 NAT and iptables persistence.
 
+Note that this role cannot perform every step necessary to set up NAT.
+Once an instance is started up, one must determine the NAT interface
+and add an iptables rule:
+
+```console
+iptables -t nat -A POSTROUTING -s "${client_network_cidr}" -o "$interface" -j MASQUERADE
```

Comment:
Did you mean to use the `${...}` and `$...` syntax here for variables?  I realize that is what you have in the [cloud-init script](https://github.com/cisagov/openvpn-server-tf-module/blob/develop/cloudinit/create-iptables-rule-for-nat.sh#L26) you wrote, but I don't know if it should be written that same way in this context.

Reply:
Good point.  See commit 76d85b5.